### ISSUE TITLE
Github identity proof not succeeding...

### ISSUE THREAD

**Issue by Loutenant4** [keybase/client#19384]
I haz no idea what I am doing wrong...
https://gist.github.com/Loutenant4/4abcc695886f7eaed550d9c970e0e177

This is the email I got from keybase:
 Hi, ljwillie4, your attempted github identity proof of @loutenant4 never succeeded. We've been checking it repeatedly, and from our perspective, it's either (a) not there, or (b) not posted precisely.

If you think there's a bug, please post an issue on github. Include the link to your proof.

**Comment by rassalas** [comment]
Yes, This is BS, they post an airdrop notice on coinbase today, and an account registered on the day of announcement alone is not good enough for the Airdrop.  I do have an old Github account, but there is no direction on how to link it.  How do you spell premined?  With this type of SNAFU, I would suggest Lumen is not worth having.

**Comment by Loutenant4** [comment]
A friend of mine had to tell me how to link it.  You go to your profile, and there are several things you can prove on the right, GitHub, Twitter, etc.  They give you text to paste and what to name the public gist, but it's saying it's not there or not posted precisely...  I think he's having the same problem though.

**Comment by Loutenant4** [comment]
also this:
https://github.com/keybase/client/issues/19361

**Comment by ActuallyStephen** [comment]
This is bullshit doesn't work and needed to be signed before the whole airdrop is announced stupid

**Comment by Loutenant4** [comment]
SOLVED:
Go to windows command prompt and type in 
keybase prove github

delete your old gist and create a new one copying and pasting from the code that the command prompt displays.

it will ask to check github; choose yes and it will work in seconds.

Note:  I could not get it to work with the gist I had already created; I had to redo it.

**Comment by rassalas** [comment]
No workey Permission denied, Make sure your gist is Public.  this shit is buggy, wtf is a Gist anyways.

**Comment by QuantumT95** [comment]
Thanks @Loutenant4 

That worked!

**Comment by nicholasmartin** [comment]
Thanks @Loutenant4 I followed exactly what you wrote (important to delete an existing keybase.md and copy from the terminal) and can confirm this works on Mac Terminal.
